drivers/storage/portworx/portworx.go

```diff
@@ -2,12 +2,15 @@ package portworx
 
 import (
 	storage "github.com/libopenstorage/operator/drivers/storage"
+	corev1alpha1 "github.com/libopenstorage/operator/pkg/apis/core/v1alpha1"
```

Comment:
alias not needed

Reply:
Just keeping it consistent in all the code